covid_model/ode_builder.py

```diff
@@ -155,35 +155,19 @@ def __init__(self, base_ode_builder=None, trange=None, attributes: OrderedDict =
         self.solution_ydf = None
 
         if base_ode_builder is not None:
-            self.params = copy.deepcopy(base_ode_builder.params)
+            if deepcopy_params:
+                self.params = {t: {cmpt: params.copy() for cmpt, params in params_by_cmpt.items()} for t, params_by_cmpt in base_ode_builder.params.items() for t in self.trange}
```

Comment:
If you do deepcopy the parameters, it takes a couple of minutes, which is why I added the option not to. That said, before I was just calling copy.deepcopy(), and it was just running and running and running. It's a lot faster to go to the depth you need to go to and copy, I guess because deepcopy has to dive in recursively to determine the depth?